api/lib/opentelemetry/instrumentation/registry.rb

```diff
@@ -7,79 +7,79 @@
 module OpenTelemetry
   module Instrumentation
     # The instrumentation Registry contains information about instrumentation
-    # adapters available and facilitates discovery, installation and
+    # instrumentations available and facilitates discovery, installation and
```

Comment:
we can probably lose this `instrumentations` from this comment
```suggestion
    # available and facilitates discovery, installation and
```

Reply:
👍 fixed